pod/perlsub.pod

```diff
@@ -473,13 +478,28 @@ that the caller passed no arguments:
 	return 123;
     }
 
-When using a signature, the arguments are still available in the special
-array variable C<@_>, in addition to the lexical variables of the
-signature.  There is a difference between the two ways of accessing the
-arguments: C<@_> I<aliases> the arguments, but the signature variables
-get I<copies> of the arguments.  So writing to a signature variable
-only changes that variable, and has no effect on the caller's variables,
-but writing to an element of C<@_> modifies whatever the caller used to
+In the current perl implementation, when using a signature the arguments
+are still also available in the special array variable C<@_>.  However,
+accessing them via this array is now discouraged, and should not be
+relied upon in newly-written code as this ability may change in a future
+version.  Code that attempts to access the C<@_> array will produce
+warnings in the C<experimental::snail_in_signatures> category when
+compiled:
+
+    sub f ($x) {
+        print "Arguments are @_";
```

Comment:
maybe consider making it clear by adding a comment? so one reader can understand the snippet without reading the paragraph above (even if he should)
```
    print "Arguments are @_"; # throws a warning
```

Reply:
Added cc1d3fb980